src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -290,17 +290,17 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
         /// <param name="requestContext"></param>
         /// <returns></returns>
         internal async Task HandleSyntaxParseRequest(SyntaxParseParams param, RequestContext<SyntaxParseResult> requestContext)
-        {            
-            await Task.Run(async () => 
+        {
```

Comment:
it often nice to separate non-function formatting changes from functional changes into different PRs.  I see you called out the relevant changes, but it's hard to spot typos, etc.

Reply:
Yeah sorry about that - I have an extension that deletes whitespace on save automatically so I don't notice the changes until I'm committing them in usually and undoing them would be a pain. If there were more actual changes I'd do that to make it clearer but since this was just 2 lines I figured it'd be fine to just call it out like I did.